tests/test_layers.py

```diff
@@ -48,15 +48,15 @@ def model_func(wrapped_import, inputs: torch.Tensor):
 
 
 def test_negative_sampling():
-  n_batch, n_time, n_feat = 3, 5, 7  # B, T', F
+  n_batch, n_time, n_feat = 3, 15, 7  # B, T', F
```

Comment:
It's probably more interesting to have a test case where `n_time % stride != 0`.